src/IdentityModel/Client/TokenClient.cs

```diff
@@ -44,9 +44,9 @@ public TokenClient(string address)
         /// or
         /// innerHttpMessageHandler
         /// </exception>
-        public TokenClient(string address, HttpMessageHandler innerHttpMessageHandler)
+        public TokenClient(string address, HttpMessageHandler innerHttpMessageHandler = null)
         {
-            if (innerHttpMessageHandler == null) throw new ArgumentNullException(nameof(innerHttpMessageHandler));
+            innerHttpMessageHandler = innerHttpMessageHandler ?? new HttpClientHandler();
```

Comment:
This follows the same pattern as https://github.com/IdentityModel/IdentityModel2/blob/dev/src/IdentityModel/Client/DiscoveryClient.cs#L116 .